linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -219,36 +211,34 @@ private[emitter] final class ClassEmitter(sjsGen: SJSGen) {
         inheritableCtorDef <-
             globalFunctionDef("h", className, Nil, None, js.Skip())
         chainProto <- chainProtoWithGlobals
-      } yield {
-        js.Block(
-            // Real constructor
-            js.DocComment("@constructor"),
-            realCtorDef,
-            chainProto,
-            genIdentBracketSelect(ctorVar.prototype, "constructor") := ctorVar,
-
-            // Inheritable constructor
-            js.DocComment("@constructor"),
-            inheritableCtorDef,
-            globalVar("h", className).prototype := ctorVar.prototype
-        )
-      }
+      } yield (
+          // Real constructor
```

Comment:
Hum this goes a bit outside the style guide. If the body of a `yield` spans multiple lines, it should have `{}`:
```suggestion
      } yield {
        // Real constructor
```

Reply:
Just that by itself will not compile because of the blank line before `// Inheritable Constructor`. The `::` before the blank line is then interpreted as postfix operator.

Options I see:
- Leave it like this.
- Curly braces without the blank line
- Parens inside the curly braces (either with nesting or on the same line).

I have gone with the last option for now.

Reply:
Hum I see. In that case ideally it should be:
```scala
} yield {
  (
    ...

    ...
  )
}
```

Reply:
Updated.